python/mxnet/torch.py

```diff
@@ -0,0 +1,157 @@
+# coding: utf-8
+"""Interface for NDArray functions executed by torch backend.
+Install torch and Compile with USE_TORCH=1 to use this module"""
+from __future__ import absolute_import
+
+import ctypes
+import sys
+from .base import _LIB
+from .base import c_array, py_str, ctypes2docstring
+from .base import mx_uint, mx_float, NDArrayHandle, FunctionHandle
+from .base import check_call
```

Comment:
Can we make this special case of general functions? It is ideal that we do not need torch.py, but ensemble these into the normal mxnet library.